cl/users/templates/profile/delete.html

```diff
@@ -29,7 +29,7 @@ <h3>This <em>cannot</em> be undone.</h3>
       <li>All of your <a href="{% url "view_visualizations" %}">network visualizations</a> ({{ non_deleted_map_count }}).</li>
     {% endif %}
     {% if request.user.user_tags.count > 0 %}
-      <li>All of your <a href="{% url "tag_list" %}">tags</a> ({{ request.user.user_tags.count }}).</li>
+      <li>All of your <a href="{% url "tag_list" username=user.username %}">tags</a> ({{ request.user.user_tags.count }}).</li>
```

Comment:
```suggestion
      <li>All of your <a href="{% url "tag_list" username=user.username %}">tags</a> ({{ request.user.user_tags.count }}).</li>
{% endif %}
{% if request.user.docket_alerts.count > 0 %}
      <li>All of your <a href="{% url "profile_alerts" %}">Docket Alerts</a> ({{ request.user.docket_alerts.count }}).</li>
```

According to the previous comment.  If it's useful to show how many docket alerts will be deleted, we should add them here.